docs/release-workflow.md

```diff
@@ -0,0 +1,20 @@
+# Release Workflow
+
+The release workflow is used for updating the GitHub release page when
```

Comment:
Change to "The motivation of release workfow is to.."

Reply:
Changed.